### PR TITLE
Add About Tab

### PR DESCRIPTION
**Pull request by lolfuljames**
About tab that includes our pictures. Clicking on it redirects to our linkedin!

On PC, hover activates the about tab:
Collapsed:
image

Expanded:
image

On Mobile, clicking the button activates the about tab, tab background is only available on the button so that it doesn't show up when result-container is open.

Collapsed:
image

Expanded:
image

Demo: https://jtan-sps-summer20.appspot.com/
Note when testing: If you use dev tools to check out the mobile site, you will have to refresh the page for the collapse function to work properly (because it was loaded from a pc-perspective initially), if you use your phone you can just ignore it.

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -166,6 +213,7 @@ <h3 id="restaurant-name">Name</h3>
 
         </div>
 
+
```
Comment (daekoon) [inline src/main/webapp/index.html]: Additional line can be removed

**Review by daekoon — COMMENTED** [review]
I feel like a user may not know what that icon means, and who the people shown by the button is. Maybe can make it more explicit, like having "developers" text that pops out as well?

**Comment by lolfuljames** [comment]
> I feel like a user may not know what that icon means, and who the people shown by the button is. Maybe can make it more explicit, like having "developers" text that pops out as well?

Good point! I've added it now

**Review by daekoon — APPROVED** [review]
LGTM!

**Pull request merged**